ixmp/reporting/utils.py

```diff
@@ -256,7 +258,7 @@ class AttrSeries(pd.Series):
     """
 
     # normal properties
-    _metadata = ['attrs']
+    _metadata = ('attrs', )
```

Comment:
out of curiosity - what is the idea behind replacing list with tuple?

Reply:
This seems to be the practice in the upstream (pandas) code base. I guess it is so that no one accidentally mutates the sequence by adding/removing items.

Reply:
https://github.com/pandas-dev/pandas/blob/master/pandas/core/generic.py#L155